### PR TITLE
add test file, some changes solidity vs code 

### PR DESCRIPTION
**Pull request by Vulturio**
Some test on PositiveTests.js file and "" to '' on import contract.

connects to #1 
connects to #6

### CODE REVIEW

File: contracts/FiatToken.sol [hunk 1]
```diff
@@ -1,9 +1,9 @@
 pragma solidity ^0.5.0;
 
-import 'openzeppelin-solidity/contracts/token/ERC20/ERC20.sol';
-import 'openzeppelin-solidity/contracts/token/ERC20/ERC20Detailed.sol';
-import 'openzeppelin-solidity/contracts/token/ERC20/ERC20Mintable.sol';
-import 'openzeppelin-solidity/contracts/token/ERC20/ERC20Burnable.sol';
+import "../node_modules/openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
```
Comment (ajunge) [inline contracts/FiatToken.sol]: This is not needed. Is solved using `.vscode/settings.json` on master branch

Idea from: https://medium.com/coinmonks/solidity-import-in-vs-code-the-right-way-82baa1cc5a71

**Review by ajunge — CHANGES_REQUESTED** [review]
Good work on the tests. No need for changes on the FiatToken.sol file. Since no contract file changed no need to change the `build/*` file

**Comment by Vulturio** [comment]
> Good work on the tests. No need for changes on the FiatToken.sol file. Since no contract file changed no need to change the `build/*` file

Sorry, the truffle compiler change the build/* file

**Review by ajunge — APPROVED** [review]
Looks good

**Pull request merged**